packages/react/core/src/ReactWallet.ts

```diff
@@ -0,0 +1,19 @@
+import type { Wallet } from '@wallet-standard/base';
+
+import type { ReactWalletAccount } from './ReactWalletAccount.js';
+import type { WalletHandle } from './WalletHandleRegistry_INTERNAL_ONLY_NOT_FOR_EXPORT.js';
+
+/**
+ * Represents a `Wallet` in a React application. It acts as a 'handle' to the underlying `Wallet`
+ * instance in the Wallet Standard registry, and contains a subset of its properties.
+ *
+ * You can pass objects of this type around your application as props or state, inspect its `chains`
+ * and `features`, and enumerate the `accounts` for which your application has been granted
+ * authorization to use.
+ */
+export type ReactWallet = WalletHandle &
+    Readonly<
+        Pick<Wallet, 'chains' | 'icon' | 'name' | 'version'> & {
+            accounts: readonly ReactWalletAccount[];
+        }
```

Comment:
Is `features` missing? The readme and docstring says you can use this to inspect features. Or are these intentionally missing here before the later PRs you mentioned in the description?

Reply:
`features` is on the `WalletHandle` type. Weird, I know.